Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst

```diff
@@ -0,0 +1,110 @@
+
+The script will: 
+
+- command LSSTComCam to take a number of bias frames,
+- call the Rubin Science Pipelines bias generation pipetask via the OCS-Controlled Pipeline System (OCPS),
+- verify the resulting bias (see the package `cp_verify`_ and `DMTN-101`_)
+- certify the resulting bias with a given range of validity dates.
+  
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _ComCam-Master-Bias-Procedure-Prerequisites:
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes  that the LSSTComcam and OCPS are both ``ENABLED``, and that latter has been ``ENABLED`` with the configuration of ``LSSTComCam``.
+- A `butler`_ is needed to run the pipetask bias command.
+
+.. _butler: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/index.html
+
+.. _ComCam-Master-Bias-Procedure-Post-Conditions:
+
+Post-Condition
+==============
+
+- A master bias image per detector will be certified in an individual butler CALIBRATION `collection`_ that can be grouped into a butler CHAINED collection to connect all of the various calibrations into a single collection.
+
+.. _collection: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/organizing.html
+
+.. _ComCam-Master-Bias-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Once you are logged into LOVE, click on the ``MTQueue`` panel, as shown in the figure below: 
+
+.. figure:: ./_static/love-mtqueue-panel.png
+    :name: MTQueue-love
+
+     Screenshot of LOVE interface with the "MTQueue" pannel.
+
+
+Load the Script
+---------------
+
+After clicking on the ``MTQueue`` panel, search for the script ``maintel/make_comcam_bias.py`` under ``AVAILABLE SCRIPTS`` on the left, as shown in the figure below:
+
+.. figure:: ./_static/love-available-scripts.png
+    :name: available-scripts-love
+
+      Screenshot of LOVE interface with the "AVAILABEL SCRIPTS" list.
+      
+Load the script by clicking on the button in front of the name of the script that has a triangle.
+
+Enter configuration parameters
+------------------------------
+
+After loading the script, a window that contains two sections, ``SCHEMA`` (top) and ``CONFIG`` (bottom), will appear. The former will show the configuration options (and the default values of some of them) available that should be entered in the latter. The configuration options are as follows:
+
```

Comment:
Can you add a text code block with a configuration people can copy and paste directly?

```
.. code-block:: text

    n_bias: 2
    detectors: (0,1,2,3,4,5,6,7,8)
    ...
```

Or at least the minumum configuration people will need (I think `n_bias` is the only required parameter right?).

Reply:
For this script, the required parameters are ``input_collections_bias, calib_collection, repo``. In the next iteration (I'm working on a script for biases, darks, and flats; DM-31306), I'll try to make in such a way that as many parameters as possible have defaults (following a suggestion by you). I'll add examples here and in the LATISS document,  as you suggest here.